### ISSUE TITLE
Datepicker dark theme broken

### ISSUE THREAD

**Issue by raimund-schluessler** [nextcloud/nextcloud-vue#121]
The date/timepicker component does not work well with the NC dark theme. The screenshots are from the Tasks vue branch.

Datepicker:
![screenshot_2018-11-25 aufgaben - nextcloud](https://user-images.githubusercontent.com/2496460/48977592-99765180-f09d-11e8-9e4e-1a8451448133.png)

Timepicker
![screenshot_2018-11-25 aufgaben - nextcloud 1](https://user-images.githubusercontent.com/2496460/48977593-9bd8ab80-f09d-11e8-9d35-48211c72db0a.png)

There are these inline rules
```
.mx-datepicker-popup {
    position: absolute;
    margin-top: 1px;
    margin-bottom: 1px;
    border: 1px solid #d9d9d9;
    background-color: #fff;
    -webkit-box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
    box-shadow: 0 6px 12px rgba(0, 0, 0, 0.175);
    z-index: 1000;
}

.mx-input-append {
    position: absolute;
    top: 0;
    right: 0;
    width: 30px;
    height: 100%;
    padding: 6px;
    background-color: #fff;
    background-clip: content-box;
}
```
where especially the `background-color: #fff;` interfere a lot.
Also in this rule
```
.mx-calendar-icon {
    width: 100%;
    height: 100%;
    color: #555;
    stroke-width: 8px;
    stroke: currentColor;
    fill: currentColor;
}
```
`color` should rather be `var(--color-main-text)`.

**Comment by skjnldsv** [comment]
How did I missed that :see_no_evil: 
Thanks a lot @raimund-schluessler ! :)

**Comment by skjnldsv** [comment]
Fixed in #193
